gitlab-ha/tests/test_login_gitlab.py

```diff
@@ -0,0 +1,57 @@
+# coding: utf-8
+import os
+import pytest
+import urllib
+from fixtures import user
+from pytest_bdd import scenario, given, when, then
+from splinter import Browser
+
+# on ajoute temporairement les drivers présents dans le dossiers drivers au path 
+DRIVERS = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'drivers/')
+for directory in os.listdir(DRIVERS):
+    os.environ['PATH'] += ":" + os.path.join(DRIVERS, directory)
+
+
+@scenario('login_gitlab.feature', 'Log in to Gitlab with admin user')
+def test_login_gitlab():
+    pass
+
+
+@given("I am an administrator user with login : <login> and password : <password>")
```

Comment:
As login is a parameter I would suggest to rename to something likes:

@given("A valid gitlab account <login> and <password>")